force-app/main/default/triggers/MedicationRequestTrigger.trigger

```diff
@@ -1,19 +1,22 @@
 /**********************************************************************************************
-* @Author:      Deepak 
+* @Author:      Accenture 
 * @Date:        14 Dec 2023
 * @Description: The purpose of this Trigger is to trigger on particular events
 * @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description]
 				19 Dec -  EDRD-282          -  Accenture   -  populate MR ExpenditureEstimate
                 09 Jan -  EDRD-139          -  Accenture   -  Calculate Drug Forecast with Respect to Case
                 09 Jan -  EDRD-139          -  Accenture   -  update Forecast On Case
 ***********************************************************************************************/
-trigger EDRD_tgr_MedicationRequest on MedicationRequest (before insert, before Update, after Update) {
+trigger MedicationRequestTrigger on MedicationRequest (before insert, before Update, after Update) {
     
     List<MedicationRequest> MRListValidate = new List<MedicationRequest>();
     
     if(trigger.isBefore){
         if(trigger.isInsert || trigger.isUpdate){
             for(MedicationRequest MRObj : trigger.new){
+               /* if(MRObj.Dosage_Units_Name__c != MRObj.Strength_Units__c){
+                    MRObj.addError(label.EDRD_label_DosageAndStrengthUnits);
```

Comment:
@deepakmulamalla - Could the commented lines of code be removed?

Reply:
Done @ckleinacn, Thanks for letting me know!